Assets/Settings.StyleCop

```diff
@@ -2,6 +2,11 @@
   <Analyzers>
     <Analyzer AnalyzerId="StyleCop.CSharp.DocumentationRules">
       <Rules>
+        <Rule Name="FileMustHaveHeader">
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
```

Comment:
You appear to be reversing some of the recent StyleCop changes. Any reason why? We actually do have headers now, and they should be enforce. Regardless, such changes should not be part of a skybox pull request.